pkg/releaseutil/manifest_sorter.go

```diff
@@ -129,7 +140,7 @@ func (file *manifestFile) sort(result *result) error {
 			return errors.Wrapf(err, "YAML parse error on %s", file.path)
 		}
 
-		if entry.Version != "" && !file.apis.Has(entry.Version) {
+		if file.validate && entry.Version != "" && !file.apis.Has(entry.Version) {
 			return errors.Errorf("apiVersion %q in %s is not available", entry.Version, file.path)
```

Comment:
I feel like a lot of these problems (and a few others we have run into) could be solved more simply by removing this check in the sorter. I am not sure why the sorter cares if it has an unsupported API version (trying to track down why that is). An unknown type should be sorted to the bottom of the list and then an actual validator should catch the missing API type. Going to do some more digging here

Reply:
I actually am leaning towards removing this check, which would solve the issue (I think). Let me whip up (yet another) PR and link it to this